diagnostics-app/package.json

```diff
@@ -7,7 +7,10 @@
     "build": "ng build",
     "watch": "ng build --watch --configuration development",
     "test": "ng test",
-    "test:ci": "ng test --watch=false --browsers=ChromeHeadless --code-coverage"
+    "test:ci": "ng test --watch=false --browsers=ChromeHeadless --code-coverage",
+    "husky:prepare": "cd .. && husky install diagnostics-app/.husky",
+    "husky:prepush": "husky add .husky/pre-push \"cd diagnostics-app\nnpm run test:ci\"",
```

Comment:
This looks a bit tricky for me. Can't we avoid using some extra package to support git hooks to make it simpler to understand what actually is running?

Also I see some duplication in github actions and here. Maybe move all logic into a script and reuse it in git hooks and github actions?

Reply:
Okay 👍🏻 I'll have a look at using inbuilt githooks.

Reply:
I removed husky as a dependency and now it uses built in git hooks!